### PR TITLE
feat: add tribes landing page

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description 

- builds out tribes landing page (only admin access )
- reorganize images for project logos + compressed them
- responsive design

screencapture-localhost-8000-tribes-2019-10-28-23_09_35

Ref: https://github.com/gitcoinco/web/issues/5419

### CODE REVIEW

File: app/retail/views.py [hunk 1]
```diff
@@ -1793,3 +1793,118 @@ def increase_funding_limit_request(request):
...
+            'text': 'I had a lot of fun (during Beyond Blockchain) meeting people and building tangible rapidly. Glad to have a winning submission!',
+            'author': 'VirajA',
+            'designation': 'Hacker',
+            'photo': 'https://c.gitcoin.co/avatars/58ef080697b34b1eab840bc60e2ee92b/viraja1.png'
+        },
+        {
+            'text': 'Gitcoin has a fantastic community that is our target audience -- Web 3 developers who want to build.',
+            'author': 'Sam Williams',
+            'designation': 'CEO, Arweave',
+            'photo': 'https://i0.wp.com/www.coininsider.com/wp-content/uploads/2018/10/Sam-Williams.jpg?resize=418%2C418&ssl=1',
```
Comment (danlipert) [inline app/retail/views.py]: we should probably get this onto our static server

File: app/retail/views.py [hunk 2]
```diff
@@ -1793,3 +1793,118 @@ def increase_funding_limit_request(request):
...
+            'org_photo': static('v2/images/project_logos/arweave.svg')
+        },
+        {
+            'text': '"Relationships with developers" is our guiding light. For both developers and ourselves, it’s great to work with GItcoin to see more working examples using Portis.',
+            'author': 'Scott Gralnick',
+            'designation': 'Co-Founder, Portis',
+            'photo': 'https://everipedia-storage.s3-accelerate.amazonaws.com/ProfilePics/__87296.png',
```
Comment (danlipert) [inline app/retail/views.py]: this one as well

**Comment by thelostone-mc** [comment]
@danlipert addressed comments ^_^

File: app/assets/v2/images/project_logos/portis.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...P7g7ADbqEgHQE+EtLupDgkAwELrn56z9oXdDxLmPWziY1MgTE8CE9lhCjfUcTqWTy50//3U3vkcQt3ICANARI6NKEAkdkJGDO2ptbW29u6Zx/Uby9tU5GH3X2yfxtPDkwlB0bGnk9l07fwmxc52gztloJkNBrJUj70BC4+Ka/+2xzR9uNzZ3zl8Wam0juHkXeXFKf6OvPJkZGDqZSU19/7u8//VcedYVZCGhFgISuVTgZjF8ELv7M1z7c1Nz0uXhr+1mNHe0NfvWrcz/m42bJgf6Xk4nk/9n2fz95l85jZWwQ8IIACd0LqtgMFYF3fPpvL2+MN3463tJ8WfPixV2hGnwNgzWEbSKbTo2mJxOPTE5N/jWnndUAk6YQMAiQ0LkNIOAyAVNQ1zyv+YKG1tZlxu/uzN5P8DVFbYmh4dTkROJgZir1Qk7ktjx366dechk/5iAQWgIk9NCGnoH7QcDcmS4Sa/rjurrYOxvb29eF7WjXxMDgcDoxvic1OfVQPpP8B/ZS9+Ouo4+wEiChhzXyjDswAubv75GI+K3Gpqa31DfPW61LkjeTd2ZqaiA7OflSKpV+nHPGA7vF6DikBEjoIQ08w5aLgPk7fDRSd1F9U3xjrL5heSwe74y3trbLdkKceeDJ1NjYiJm4M+nUgWwq9VQ2l/slIja57ie8CScBEno4486oFSIwPaPPR5bU1UfPicbqTzddb5zXen5hCNG6uuZalfaTwyOpXDabKNicHB970fycy6Rfz6ZzL+cj+cMk7QId3iEAAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQAACEIAABCAAAQhAoDoC/z/xLpz8FGhLcgAAAABJRU5ErkJggg==" transform="translate(-569 -4800)" fill="none" fill-rule="evenodd"/></svg>
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/assets/v2/images/project_logos/portis.svg]: @octavioamu I didn't get what that comment means 😅 
What do you mean when you say look for svg logo ?

File: app/assets/v2/images/project_logos/status.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...VqfE/intIuZD9EOPVzRxUCo6rt+d2XvohuGPfviC1cv0rh7NTpofOjWOsAjSmBG70MgMieFSJlTUshQTILEPLAIWAYtAhxGYnyxFpZIRV4BHThBLaK4C8vDKPnXLraVrr7+7lFmRDzIcURn4oQocnKFelqWoYDhnfw1LYXI3Jq2EqxDKrhkGuECPZksUvXTuzFM3vfvta7rgG/XZYDoO4fQGEE4zJyytb1mKOWtHm7FFwCJgEWgWgfnNUlALoSVcK8QIIwajJSwsagn3vf+p/uFff/HCC4HrrPLcLMsl0YKH5i2smz8IaMmEkR9ocUWA+AEb2z2Zse6u4Y/9/lnr16m+tBrL5Xsz6Xyx5Hlly+vjLMXEihhBhdHn5Z1J2YQZ9+nb8TvpPiZI1WU8RNVje2MRsAhYBCwCLSMw71kKaiST1GieytoHBhYNwzCAuGKv+u53X7zvNy/tGwwTqR6sVsiGkdBlC6p18wYBWAqO5IAPcEoJlUuGHNsxcOF5W6+8fFk6pYKSSiXLbUqJIf+seiXiPKG0frw2lqWIo2H9FgGLgEVgHiCwcFgKwIroCmsfjr5jMQT2AnnFr+9T//ubPxkc7HcSK1/ZPZBMdXFQRFil2jcPwN5fihBJBMoigHw+n8lwXP1Yb3c2KOZyIzs2HOh8+P2nHb7FWMMUAyQIHJwyG1ERSMXwipo+elZmKaL7sqda6jB+F5WpJsJ4iJoX9tYiYBGwCFgEWkRggbAU1AqiEtEVmb/CVYR69b0AuUhq9uK6H4599/u/CN2+RLJ/NB8GjmeXP1rsDx0JLuSba/mw0EIx392DgcxRJxhKB7krLz39zRd5NFgqgTUzbRDTOE3c44S/ithHTV8ODB+pw8bDy5uqWHbhI4LLeiwCFgGLwMwjsEBYiiqKAh1hPb7i3FQQBGNBkPa8YaU3g3zzmicfemRP0V9aKGpqNSVXYZQH9RTZukYImL0bRvWVUNWEu040NnfQRkiTutjZUQoGHW+o5Lx4wtZ1H3vniRuXqTwNijpFYrItnzrFqjyqOoDJz7IUdWC3jywCFgGLwFwisNBYCghJaPYOlGkMdIepcFB0Sr6bHik5oetyxNQtt6v/d+sTL+0YKRTZN+CFDqdXCoXS2xoN8zA+v8UUNC1gWYpG3bCsXBnftUFwDX65HTBjyuYMx/MxJUFgpxS67PbFtilKL6Vs
```
Comment (octavioamu) [inline app/assets/v2/images/project_logos/status.svg]: this one is missing

File: app/assets/v2/images/project_logos/portis.svg (hunk 3 above)
Comment (octavioamu) [inline app/assets/v2/images/project_logos/portis.svg]: this one is missing

**Pull request merged**